backend/src/test/java/ai/verta/modeldb/FindProjectEntitiesTest.java

```diff
@@ -1696,13 +1697,9 @@ public void findExperimentRunPredicateValueEmptyNegativeTest() {
             .addAllPredicates(predicates)
             // .setIdsOnly(true)
             .build();
-    try {
-      experimentRunServiceStub.findExperimentRuns(findExperimentRuns);
-      fail();
-    } catch (StatusRuntimeException exc) {
-      Status status = Status.fromThrowable(exc);
-      assertEquals(Status.INVALID_ARGUMENT.getCode(), status.getCode());
-    }
+    AdvancedQueryExperimentRunsResponse response =
+        hydratedServiceBlockingStub.findHydratedExperimentRuns(findExperimentRuns);
+    assertEquals("Expected response not found", 0, response.getHydratedExperimentRunsCount());
```

Comment:
Now we are checking permission first so if given project id is not valid it will return empty list instead of PERMISSION_DENIED